src/pcapi/core/users/api.py

```diff
@@ -205,7 +204,7 @@ def create_account(
 def steps_to_become_beneficiary(user: User) -> list[BeneficiaryValidationStep]:
     missing_steps = []
 
-    if feature_queries.is_active(FeatureToggle.FORCE_PHONE_VALIDATION) and not user.is_phone_validated:
+    if FeatureToggle.FORCE_PHONE_VALIDATION.is_active() and not user.is_phone_validated:
```

Comment:
Idem ici, inversons l'ordre des expressions dans la condition. Oh la la, comment on optimise tout comme des brutes !

```suggestion
    if not user.is_phone_validated and FeatureToggle.FORCE_PHONE_VALIDATION.is_active():
```